### ISSUE TITLE
Sample bookinfo product page doesn't load Glyphicons fonts

### ISSUE THREAD

**Issue by PaulMilla** [istio/istio#13153]
**Describe the bug**

The latest docker image (1.11.0) for the sample bookinfo product page has switched (#12646) to using a packaged bootstrap.min.css introduced in #12218 which defines the Glyphicons Halflings font as follows: 

```css
@font-face {
    font-family: 'Glyphicons Halflings';
    src: url(../fonts/glyphicons-halflings-regular.eot);
    src: url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')
}
```

The problem with this is that the docker image doesn't contain the referenced font files nor are they checked in to the repo and thus the star & user fonts appear as broken font figures

**Expected behavior**

Running the latest [istio/examples-bookinfo-productpage-v1](https://hub.docker.com/r/istio/examples-bookinfo-productpage-v1/) docker image produces a site without broken glyphicon fonts

**Steps to reproduce the bug**

- `docker run -p 9080:9080 istio/examples-bookinfo-productpage-v1:1.11.0`
- navigate to http://localhost:9080/productpage
- check console for errors
  - If `istio/examples-bookinfo-reviews-v2` or `istio/examples-bookinfo-reviews-v3` backend is setup then broken fonts will appear under the Book Reviews section
  - If no other backend is setup then a broken user font will appear after signing in using the "Sign in" button at the top right

**Version**

```shell
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.11", GitCommit:"637c7e288581ee40ab4ca210618a89a555b6e7e9", GitTreeState:"clean", BuildDate:"2018-11-26T14:38:32Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.11", GitCommit:"637c7e288581ee40ab4ca210618a89a555b6e7e9", GitTreeState:"clean", BuildDate:"2018-11-26T14:25:46Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Installation**

N/A

**Environment**

Docker-Desktop (v2.0.0.3-stable) for Windows 10 Enterprise 2016 LTSB

**Comment by rvennam** [comment]
Should be fixed with 1.12.0

**Comment by thimmaiah** [comment]
Broken again in 1.15.0
Why would the css not be loaded ?
